README.md

```diff
@@ -286,8 +286,7 @@ be used from disk if required.
 Metadata scripts implement support for running user provided
 [startup scripts](https://cloud.google.com/compute/docs/startupscript) and
 [shutdown scripts](https://cloud.google.com/compute/docs/shutdownscript). The
-guest support for metadata scripts is implemented in Python with the following
-design details.
+guest support for metadata scripts consider the following details:
```

Comment:
Can you also add this one line - 

Refer [Use startup scripts on Windows VMs](https://cloud.google.com/compute/docs/instances/startup-scripts/windows) for Windows specific details.

Reply:
done.